apps/aevm/src/aevm_eeevm_state.erl

```diff
@@ -195,10 +195,10 @@ is_reentrant_call(State) ->
 
 %% TODO: Currently the Store is saved both in the chain state and the VM state.
 %%       Refactor?
-get_store(#{chain_api := ChainAPI, chain_state := ChainState}) ->
-    ChainAPI:get_store(ChainState).
+get_store(#{storage := Store}) -> Store.
```

Comment:
I need to understan the changes in this module. The rest seems ok.

Reply:
The change is to remember the store in the VM state, rather than to compute it from state trees every time we need it.

This is ok, since we don't write to the store during contract execution (no reentrant calls in Sophia).